### PR TITLE
chore(test): split single test into multiple tests

### LINKED ISSUE #10: Problem with having a single unit test
Currently unit tests for `get-func-name` are so tiny and simple that I can just copy and paste it over here 😄 
```js
'use strict';
var assert = require('simple-assert');
var getFuncName = require('..');
describe('getFuncName', function () {
  it('getFuncName', function () {
    // Asserting that `getFuncName` behaves correctly
    function /*one*/correctName/*two*/() { // eslint-disable-line no-inline-comments, spaced-comment
      return 0;
    }
    function withoutComments() {
      return 1;
    }

    var anonymousFunc = (function () {
      return function () { // eslint-disable-line func-style
        return 2;
      };
    }());
    assert(getFuncName(correctName) === 'correctName');
    assert(getFuncName(withoutComments) === 'withoutComments');
    assert(getFuncName(anonymousFunc) === '');
  });
});
```
I know this is beautiful, but...

- If the test fails, we wont be able to tell what's going on just by reading the message.
screen shot 2016-10-11 at 9 20 48 pm
What have I broke here? Imagine if this fails only in IE running over saucelabs 😸.

- If this keeps growing I'm afraid that this can become very confusing like the [utilities tests](https://github.com/chaijs/chai/blob/cc79e11b511ecbb54e78b1b2308157ee75ee386e/test/utilities.js) of chaijs/chai.

As @meeber suggested [here](https://github.com/chaijs/chai/pull/799#issuecomment-247973900), I think that we should refactor this test and split it so it has a single assertion per test and better names.
I understand that `get-func-name` should not grow a lot and will probably stay very simple, but I don't see this as a good reason to not have good descriptive tests.

If you guys agree. I'll be very happy to make a PR addressing this.

### PR DESCRIPTION
**Pull request by vieiralucas**
Hello there,

This aims to fix #10  

Please give me your opinions on the messages

### CODE REVIEW

**Comment by meeber** [comment]
me like

File: test/index.js [hunk 1]
```diff
@@ -2,22 +2,30 @@
...
+  it('should get the function name', function () {
+    function normalFunction() {
+      return 1;
+    }
+
+    assert(getFuncName(normalFunction) === 'normalFunction');
+  });
+
+  it('should get correct name when function is surrounded by comments', function () {
     // Asserting that `getFuncName` behaves correctly
```
Comment (vieiralucas) [inline test/index.js]: Fixed

**Comment by meeber** [comment]
LGTM!

**Comment by lucasfcosta** [comment]
LGTM too!
Github reactivated the `outdated diff` feature, oh god this feels so good.

**Comment by vieiralucas** [comment]
YES, I noticed that.
I'm so happy 😄 
Github is awesome :heart:

**Pull request merged**